src/index.ts

```diff
@@ -1,37 +1,19 @@
-import { connectDb } from './app';
+import '@extensions/zod/register';
+import { create as createApp } from './app';
 
-connectDb().catch(console.dir);
+async function bootstrap() {
+  // create the app
+  const app = await createApp();
 
-// import { createHttpTerminator } from 'http-terminator';
-// import { create as createApp } from './app';
+  // start server
+  try {
+    await app.listen({ port: 5050, host: '0.0.0.0' });
```

Comment:
Refactor the app so the port could be configurable (e.g. taken from the `.env` file) not hardcoded. You can still have a default value.